### PR TITLE
Remove unused code

### PR DESCRIPTION
**Pull request by mgdelacroix**

### CODE REVIEW

**Review by esarafianou — APPROVED** [review]

File: server/plugin.go [hunk 1]
```diff
@@ -504,7 +504,7 @@ func generateSecret() (string, error) {
 }
 
 func (p *Plugin) GenerateRandomPassword() string {
-	lowerCharSet := "abcdedfghijklmnopqrst"
+	lowerCharSet := "abcdefghijklmnopqrstuvwxyz"
```
Comment (lieut-data) [inline server/plugin.go:507]: Should we just remove this function altogether? It looks to be unused now.

**Review by lieut-data — APPROVED** [review]

File: server/plugin.go (hunk 1 above)
Comment (mgdelacroix) [reply server/plugin.go:507]: 0/5 really, if we keep it there, it should be fixed, but it does look like dead code by now. If we can't think of any other use for this coming in the near future, I'm happy to delete it

Comment (lieut-data) [reply server/plugin.go:507]: Yeah, let's just delete it. Easy to bring back if/when we need to: the goal for v10 is to reduce the surface area for @esarafianou and team and streamline security review.

**Review by lieut-data — APPROVED** [review]

**Review by esarafianou — APPROVED** [review]

**Review by jespino — APPROVED** [review]
LGTM

**Pull request merged**